bundle/language_it.properties

```diff
@@ -890,6 +890,7 @@ angal.menu.btn.opd
 angal.menu.btn.opdchart                                                                                = Report Visita
 angal.menu.btn.operation                                                                               = Interventi
 angal.menu.btn.operationtype                                                                           = Tipologie Intervento
+angal.menu.btn.operationlist                                                                           = Lista Interventi
```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.